packages/maskbook/src/plugins/Transak/define.tsx

```diff
@@ -1,27 +0,0 @@
-import { PluginConfig, PluginStage, PluginScope } from '../types'
-import { PLUGIN_IDENTIFIER } from './constants'
-import { BuyTokenDialog } from './UI/BuyTokenDialog'
-
-export const TransakPluginDefine: PluginConfig = {
-    id: PLUGIN_IDENTIFIER,
-    pluginIcon: '💸',
-    pluginName: 'Transak',
-    pluginDescription: 'The Fiat On-Ramp Aggregator on Mask Network.',
-    identifier: PLUGIN_IDENTIFIER,
-    stage: PluginStage.Production,
-    scope: PluginScope.Public,
-    PageComponent() {
-        return (
-            <>
-                <BuyTokenDialog />
-            </>
-        )
-    },
-    DashboardComponent() {
-        return (
-            <>
-                <BuyTokenDialog />
-            </>
-        )
-    },
```

Comment:
If you need it to be responsive in the dashboard, you'll need to add an entry of the dashboard for this plugin.

Reply:
What if the `<BuyTokenDialog />` is controlled as a `RemoteControlledDialog`?

Reply:
It will need to have a dashboard entry, otherwise it won't response.

Reply:
The remote-controlled dialogs need components to be injected into documents before using them.

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/3496/commits/f193092eec774ff15b9f6ce025e82c0f637e8e3b